src/main/java/seedu/address/model/person/Appointment.java

```diff
@@ -99,10 +107,22 @@ public void unmark() {
         this.isMarked = false;
     }
 
+    public void setPatient(Person patient) {
+        this.patient = patient;
+    }
```

Comment:
Would it be sufficient if we kept a reference to the patient name instead? Then we can store it in the Json without needing to iterate thru all appointments to set the patient during conversion.

Reply:
Could it work if modify the constructor to take in the appointment's owner, so we don't have to manually set it all the time?